### PR TITLE
Remove product from wishlist when the product (or the combination) is removed

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Remove product from wishlist when the product (or the combination) is removed
| Type?             | bug fix
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes PrestaShop/PrestaShop#28130
| How to test?      | Add product or combination to a wishlist<br>Delete the product or a combination from the backoffice<br>The statistics page runs well.

### CODE REVIEW

File: blockwishlist.php [hunk 1]
```diff
@@ -198,6 +202,48 @@ public function hookDisplayProductActions(array $params)
...
+    }
+
+    public function hookActionProductAttributeDelete(array $params)
+    {
+        if (!isset($params['id_product']) || !isset($params['id_product_attribute'])) {
+            return;
+        }
+
+        // Remove all attributes from a product
+        if (isset($params['deleteAllAttributes']) && $params['deleteAllAttributes']) {
```
Comment (PierreRambaud) [inline blockwishlist.php:222]:
```suggestion
        if (!empty($params['deleteAllAttributes'])) {
```

**Review by PierreRambaud — APPROVED** [review]
one not mandatory comment but it's ok for me

File: blockwishlist.php (hunk 1 above)
Comment (Progi1984) [reply blockwishlist.php]: Fixed

**Review by AureRita — APPROVED** [review]
Hi @Progi1984 

Thank you for your PR, I tested it and it works, as we can see here :

https://user-images.githubusercontent.com/100699445/163371207-4035d508-f033-45d4-b30c-8c78b4ede55d.mp4

**Comment by Progi1984** [comment]
Thanks @AureRita

**Pull request merged**